NEWS.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.6 (2020-12-22)
+
+### Features
+
+- OmegaConf now supports `int` for dictionary key types ([#149](https://github.com/omry/omegaconf/issues/149)) 
+
```

Comment:
sorry for not being more specific.
Do not edit the news file.

OmegaConf is using towncrier to generate this file. You can read about the general approach in this Hydra page: https://hydra.cc/docs/development/documentation#news-entries

You should create a specific file under the news directory.

Reply:
Ok, got it.